tests/test_compose.py

```diff
@@ -0,0 +1,28 @@
+import numpy as np
+
+from videopython.base.compose import VideoComposer
+from videopython.base.transforms import TransformationPipeline
+from videopython.base.transitions import FadeTransition, InstantTransition
+
+
+def test_vanilla_compose(small_video):
+    videos_to_compose = [small_video, small_video]
```

Comment:
I would change this to two different videos so we can detect potential issues when only one video is used. In the current setting, the last frame will still be the same in such cases.